photon-server/src/main/java/org/photonvision/common/hardware/HardwareManager.java

```diff
@@ -91,6 +91,18 @@ public void shutdown() {
         LEDs.values().forEach(GPIOBase::shutdown);
     }
 
+    public GPIOBase redStatusLED() {
+        return LEDs.get(hardwareConfig.statusRGBPins.get(0));
+    }
+
+    public GPIOBase greenStatusLED() {
+        return LEDs.get(hardwareConfig.statusRGBPins.get(1));
+    }
+
+    public GPIOBase blueStatusLED() {
+        return LEDs.get(hardwareConfig.statusRGBPins.get(2));
+    }
+
```

Comment:
Will this cleanly handle a value missing from the array?
i.e. `[ 11, 12 ]` instead of `[ 11, 12, -1] ]`